public/js/app/hooks/useRoundUtils.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useLanguage } from './useLanguage'
+import { useMissingInfo } from './useMissingInfo'
+
+export const useRoundUtils = () => {
```

Comment:
Maybe `useRoundLabel` `useRoundHeader` instead of `useRoundUtils`? Lets discuss this

Reply:
```
export const useRoundLabel = round => {
  const { translation } = useLanguage()
  const { isMissingInfoLabel } = useMissingInfo()

  const { round_short_name, round_funding_type, round_category } = round
  let fundingType
  if (round_funding_type === 'PER' || round_funding_type === 'UPP' || round_funding_type === 'SAP') {
    fundingType = translation.courseRoundInformation.round_type[round_funding_type]
  } else {
    fundingType = translation.courseRoundInformation.round_category[round_category]
  }
  return ` ${!isMissingInfoLabel(round_short_name) ? `${round_short_name}` : ''} ${fundingType}`
}

```